pkg/diagnose/firewall.go

```diff
@@ -45,6 +45,9 @@ const (
 
 const (
 	clientSourcePort = "9898"
+	loadBalancerName = "submariner-gateway"
```

Comment:
It seems we can use the `names.GatewayComponent` constant rather than defining a new one.

Reply:
Not sure about that, GW LB constants defined [here](https://github.com/submariner-io/submariner-operator/blob/devel/controllers/submariner/loadbalancer_resources.go#L34-#L37) , I think we can either update operator code to expose this constants (preferable option)  or leave it as is.

Reply:
yeah that sounds fine.